config/phenomenal_tools2container.yaml

```diff
@@ -479,6 +480,13 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - mtbls-labs-uploader
+- docker_image_override: isatab-create
+  docker_owner_override: phnmnl
+  docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_tag_override: latest
```

Comment:
Yep, we need a release tag for isatab-create here please.

Reply:
Yep, on it's way shortly!